latch/resources/workflow.py

```diff
@@ -1,7 +1,9 @@
 import inspect
+import sys
+import typing
```

Comment:
This import is specifically for `typing._GenericAlias`, as 1) I prefer to avoid importing private attributes directly and 2) I wanted to avoid potential collisions with `typing.GenericAlias` or `types.GenericAlias` in case someone accidentally removed the leading underscore